app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { getCategories } = require("./controllers/categories");
+const app = express();
+
+app.use(express.json());

```

Comment:
This should not be included as we have no use of the middle-ware yet. Please remove before merging.

Reply:
ok thanks